test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/020_ATF_P_TC_navi_app_PTU_no_certificate_ForceProtectedService_OFF_RPC_encrypted_true.lua

```diff
@@ -0,0 +1,166 @@
+end
+
+function Test:Precondition_PTU_NO_certificate()
+  local SystemFilesPath = commonFunctions:read_parameter_from_smart_device_link_ini("SystemFilesPath")
+
+  EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate", {status="UP_TO_DATE"})
+
+  local RequestId_GetUrls = self.hmiConnection:SendRequest("SDL.GetURLS", { service = 7 })
+  EXPECT_HMIRESPONSE(RequestId_GetUrls,{result = {code = 0, method = "SDL.GetURLS"} } )
+  :Do(function(_,_)
+      self.hmiConnection:SendNotification("BasicCommunication.OnSystemRequest",
+        { requestType = "PROPRIETARY", fileName = "PolicyTableUpdate"})
+      EXPECT_NOTIFICATION("OnSystemRequest", {requestType = "PROPRIETARY"})
+      :Do(function()
+          local CorIdSystemRequest = self.mobileSession:SendRPC("SystemRequest", {requestType = "PROPRIETARY", fileName = "PolicyTableUpdate"},
+          "files/ptu_certificate_exist.json")
+          EXPECT_HMICALL("BasicCommunication.SystemRequest",{ requestType = "PROPRIETARY", fileName = SystemFilesPath.."/PolicyTableUpdate" })
+          :Do(function(_,_data1)
+              self.hmiConnection:SendResponse(_data1.id,"BasicCommunication.SystemRequest", "SUCCESS", {})
+              self.hmiConnection:SendNotification("SDL.OnReceivedPolicyUpdate", { policyfile = SystemFilesPath.."/PolicyTableUpdate"})
+            end)
+          EXPECT_RESPONSE(CorIdSystemRequest, { success = true, resultCode = "SUCCESS"})
+        end)
+    end)
+end
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+
+function Test:TestStep_RPC_ACK_encrypt_false()
+  self.mobileSession.correlationId = self.mobileSession.correlationId + 1
+
+  local msg = {
+    serviceType = 7,
+    frameInfo = 0,
+    rpcType = 0,
+    rpcFunctionId = 48,
+    encryption = true,
+    rpcCorrelationId = self.mobileSession.correlationId,
+    payload = '{ "audioStreamingIndicator" : "PAUSE" }'
+  }
+  self.mobileSession:Send(msg)
+
+  local startserviceEvent = Event()
+  startserviceEvent.matches =
+  function(_, data)
+    return ( data.frameType == 0 and data.serviceType == 7)
+  end
+
+  self.mobileSession:ExpectEvent(startserviceEvent, "Service 7: StartServiceACK")
+  :ValidIf(function(_, data)
+      if data.frameInfo == 2 then
+        if(data.encryption == true) then
+          commonFunctions:printError("Service 7: StartService ACK, encryption: true is received")
+          return false
+        else
+          print("Service 7: StartServiceACK, encryption: false")
+          return true
+        end
+      elseif data.frameInfo == 3 then
+        commonFunctions:printError("Service 7: StartService NACK is received")
+        return false
+      else
+        commonFunctions:printError("Service 7: StartServiceACK/NACK is not received at all.")
+        return false
+      end
+    end)
+
+  EXPECT_HMICALL("UI.SetAudioStreamingIndicator", {}):Times(0)
+  EXPECT_RESPONSE(msg.rpcCorrelationId, { success = false, resultCode = "REJECTED" })
```

Comment:
@dboltovskyi, 
1. why message is rejected in case of StartService ACK
Message is not rejected, it's expected result code from SDL. Because used RPC is SetAudioStreamingIndicator, it will REJECTED in case of any app different than media. Only media application will get SUCCESS.

2. why do we need such checks (149 and 150) - these are not described in req-t
Based on point 1. it's verify that SDL will process RPC as expected: for non-media applications (default, navi, voice-comm) SDL should not process request to HMI, but reply to mobile request.